src/concepts/choosing_a_dataset_mobile.md

```diff
@@ -0,0 +1,96 @@
+# Choosing a Dataset - Mobile Edition
+
+## Products Overview 
+
+Before doing an analysis, it is important to know which products you want to include. Here is a quick overview of Mozilla's mobile products. 
+
+
+| Product Name           | App Name           | OS      | Notes                | 
+| ---------------------- | ------------------ | ------- | -------------------- |
+| Firefox Android        | `Fennec`           | Android |                      |
+| Firefox iOS            | `Fennec`           | iOS     |                      |
+| Focus Android          | `Focus`            | Android |                      |
```

Comment:
Can you add a brief description or link to a description for Focus?

Reply:
Done